docs/sphinx/index.rst

```diff
@@ -1,50 +1,33 @@
-* `Pytest <https://docs.pytest.org/en/latest/>`_ testing framework
-* Continuous Integration with :ref:`Travis <travis-section>` and `Coveralls <https://coveralls.io/>`_
-* :ref:`Pip <deploying-section>`-ready product
-* :ref:`Sphinx Documentation <sphinx-section>` with :ref:`Read The Docs <rtd-section>` integration
-* Versioning with :ref:`BumpVersion <bumpversion-section>`.
-* :ref:`Invoke <invoke-section>` for shell tasks
-* SDSS-compliant `license file <https://github.com/sdss/python_template/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/LICENSE.md>`_.
-* `Module file <https://github.com/sdss/python_template/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/etc/%7B%7Bcookiecutter.package_name%7D%7D.module>`_.
-* :ref:`Configuration file <conf-log-section>` and improved :ref:`logging <conf-log-section>`.
-* the SDSS :ref:`tree and sdss_access <sdsspy>` python packages.
 
-Directory Contents
-------------------
+.. contents:: **Table of Contents**
 
-* **cextern**: The directory for placing C code to be compiled
-* **docs**: The directory for Sphinx documentation and other docu-related files
-* **etc**: The directory containing your SDSS modulefile and other etc
-* **python**: Your new python package directory
-* **python/package_name/core**: A directory for high-level core classes used in your product.  Contains a set of custom python Exceptions.
-* **python/package_name/etc**: An etc directory with text files that will be installed with the product. Contains a YAML configuration file that is ready by the package when imported.
-* **python/package_name/utils**: General-use tools, including a custom logger and colour printing routines.
-* **python/package_name/tests**: The directory containing the tests for the package. Includes a ``conftest.py`` file with basic configuration using `pytest <https://docs.pytest.org/en/latest/>`_.
-* **CHANGELOG.rst**: A file documenting changes to your code, e.g. new features, fixed issues, or bug-fixes.
-* **CODEOWNERS**: A file assigning ownership of the code to the package or components of the package to various users
-* **README.rst**: A file describing your package.  This will be the main display on Github.
-* **STYLE.rst**: The SDSS style guide for best coding practices.
-* **LICENSE.md**: The open source license for your product.  DO NOT DELETE.
-* **setup.py**: The setup for your pip-deployable product.  Also used when installing manually with `python setup.py install`.
-* **tasks.py**: A list of all invoke tasks available.
-* **requirements[_xxx].txt**: These files list all Python packages that are dependencies for your product.  Needed by pip.
-* **readthedocs.yml**: The configuration file for Read The Docs.
-* **.travis.yml**:  The configuration file for Travis CI.
-* **.bumpversion.cfg**: The configuration file for Bumpversion.
-* **.coveragerc**: The configuration file for python code coverage and Coveralls.
+
+What you get with this template
+-------------------------------
```

Comment:
Since this template is quite different then v1, I would add a section to the documentation after this one one "Major Changes from v1" and briefly list them.  You sort of cover this in the Changelog and in parts of this home page but I think a dedicated section would help clarify.  It took me awhile to notice how the `bumpversion` and `invoke` changes were implemented.  This doesn't have to include all new features moving forward but maybe just the big structural ones from v1.

Reply:
If we're planning on keeping the `python-template-v1` branch around, do we want to include a blurb on how to create a new package with it, just in case, a la `cookiecutter https://github.com/sdss/python_template.git --checkout python-template-v1`?  I don't know why anyone would want to do, but you never know.

Reply:
Done in both cases.